gems/aws-sdk-s3/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 Unreleased Changes
 ------------------
+* Issue - Add if_match to multipart download.
```

Comment:
I would be more descriptive on what has been changed. The CHANGELOG entry should be easy to understand without knowing any context for the customer's POV